src/main/java/com/onarandombox/MultiverseCore/MultiverseCore.java

```diff
@@ -124,6 +127,9 @@ public void onEnable() {
             config.setFirstSpawnLocation(firstSpawnWorld.getName());
         }
 
+        var newWorldManager = newWorldManagerProvider.get();
+        newWorldManager.loadAllWorlds();
```

Comment:
can `loadAllWorlds` just be part of the constructor for the new world manager?

Reply:
maybe? but i need to ensure the load order first, which is not the priority rn, so I just did the same as before